numpy/linalg/tests/test_linalg.py

```diff
@@ -1481,9 +1483,7 @@ def test_bad_args(self):
         # Using `axis=<integer>` or passing in a 1-D array implies vector
         # norms are being computed, so also using `ord='fro'`
         # or `ord='nuc'` raises a ValueError.
-        assert_raises(ValueError, norm, A, 'fro', 0)
```

Comment:
Hmmm - the inclusion of these tests implies that the failure of `ord='fro'` was intentional for 1D vectors. I'm not sure why this would be the case, as the default behavior (i.e. `ord=None`) is equivalent to `ord='fro'`. I'm no expert in vector math, but [that wolfram article](http://mathworld.wolfram.com/FrobeniusNorm.html) originally cited by @TNonet makes it seem that the equivalence between "Frobenius norm" and the L2 vector norm is colloquially accepted but perhaps not strictly true. Maybe @seberg remembers if there was ever conversation along these lines related to #14719 etc.?

Either way, my vote would be for keeping these proposed changes, as they reflect the behavior of the code (i.e. default is equivalent to Frobenius even for the 1D case).

Reply:
It seems that [MathWorks](https://www.mathworks.com/help/matlab/ref/norm.html), implements the Frobenius Norm for an m by n matrices where m and n are both greater or equal to 2. In addition, they directly implement [VectorNorms](https://www.mathworks.com/help/matlab/ref/vecnorm.html) which doesn't have Frobenius Norms.

However, [NetLib](https://www.netlib.org/lapack/lug/node75.html) states the 2-norm and the Frobenius Norms are equivalent for vectors.

It seems like there is no standard.

Therefore, I think that as long as no downstream code is adversely affected, I agree with @rossbar that the change is reflected in the behavior of the code.

Reply:
OTOH we certainly don't want to promote incorrect/imprecise use of terminology. In the absence of definitive, unambiguous resolution of the terminology, it is probably best to keep the current behavior but to clean up the docstring and the error messages so that the distinction between Frobenius (relevant ONLY for matrices) and L2-norm (default behavior for vectors) explicit.

Reply:
I am not quite sure, I tend to: Leaving it as is seems good, but the error message should be clear. Note that there is a nice table in the `Notes` section of the docstring, which is very clear on the implemented meaning.